### PR TITLE
Extend some commercial switches

### PR DESCRIPTION
**Pull request by JonNorman**
These switches need to be audited I think - we have quite a few and I'm fairly sure we haven't toggled any of them in months...

### CODE REVIEW

**Review by lps88 — APPROVED** [review]

File: common/app/conf/switches/CommercialSwitches.scala [hunk 1]
```diff
@@ -261,7 +261,7 @@ trait CommercialSwitches {
     description = "Solicit feedback for ad impressions",
     owners = Owner.group(SwitchGroup.Commercial),
     safeState = Off,
-    sellByDate = new LocalDate(2017, 7, 12),
+    sellByDate = new LocalDate(2017, 7, 26),
```
Comment (janua) [inline common/app/conf/switches/CommercialSwitches.scala]: 26th?

**Comment by janua** [comment]
:+1:

File: common/app/conf/switches/CommercialSwitches.scala (hunk 1 above)
Comment (JonNorman) [inline common/app/conf/switches/CommercialSwitches.scala]: You said two weeks no?

Comment (janua) [inline common/app/conf/switches/CommercialSwitches.scala]: End of the month 🤓 😄 
I guess we will have decided by then!!

**Comment by JonNorman** [comment]
[Ad Feedback Test Expires Tomorrow - Extend or remove?](https://trello.com/c/i5TQjYNw/106-ad-feedback-test-expires-tomorrow-extend-or-remove)

**Comment by JonNorman** [comment]
[keep-video-ad-slots-open ab test expiring tomorrow](https://trello.com/c/6fToAgzd/107-keep-video-ad-slots-open-ab-test-expiring-tomorrow)

**Pull request merged**